db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+user1 = User.create(name: "chris", email: "chris@chris.com", password: "123")
```

Comment:
be careful with these legit email addresses.  occasionally rails emits a real deal email and may / may not spam these test individuals who will sue you.  :fearful: